[32mINFO    [0m root:test_time_series_daily.py:37 test_time_series_daily_invalid_symbol started
[32mINFO    [0m root:test_time_series_daily.py:39 url : https://www.alphavantage.co/query?function=TIME_SERIES_DAILY&symbol=invalid&apikey=0S5QXIPXLA3EYXH5
[32mINFO    [0m root:test_time_series_daily.py:42 response : {
    "Error Message": "Invalid API call. Please retry or visit the documentation (https://www.alphavantage.co/documentation/) for TIME_SERIES_DAILY."
}
[32mINFO    [0m root:test_time_series_daily.py:47 test_time_series_daily_invalid_symbol ended